### DISCUSSION TITLE
Multi USRP X310 do not connect simultaneously

### DISCUSSION THREAD

**Discussion by Erick5G** [srsran/srsRAN_Project#895] [General Help]
Hello @pgawlowicz 
I'm trying to configure two x310 together with multi_usrp.py. I already registered that both link ports were active. There are no compilation errors.

Following some instructions from https://files.ettus.com/manual/page_multiple.html. I created a python script to add the multi_usrp. and it was achieved. Attachment

# Configure IP addresses for the two USRP devices
args = "addr0=192.168.40.2,addr1=192.168.30.2"
usrp = uhd.usrp.MultiUSRP(args)

# Configure master clock and synchronization
usrp.set_master_clock_rate(200e6) # Switch to 200 MHz
usrp.set_time_next_pps(uhd.types.TimeSpec(0.0))

usrp.set_rx_subdev_spec(uhd.usrp.SubdevSpec("A:0"), 0) # Channel A on the first radio
usrp.set_tx_subdev_spec(uhd.usrp.SubdevSpec("A:0"), 0) # Channel A on the first radio radio

usrp.set_rx_subdev_spec(uhd.usrp.SubdevSpec("A:0"), 1) # Channel A on the second radio
usrp.set_tx_subdev_spec(uhd.usrp.SubdevSpec("A:0"), 1) # Channel A on the second radio

But when I'm testing my gnb.yml only one x310 radio connects and the other one doesn't activate. They keep the PPS and ref sync, as well as the link when the gnb is initialized.

Any suggestions on what could be wrong?
[multi_x310_gnb.txt](https://github.com/user-attachments/files/17500891/multi_x310_gnb.txt)

**Comment by pgawlowicz** [comment]
Hi, multi-USRP is not supported.
